extension/rikaicontent.ts

```diff
@@ -679,6 +685,11 @@ class RcxContent {
     return text;
   }
 
+  #isElementVisible(element: HTMLElement) {
```

Comment:
Using JS private field might not make sense in typescript since `private` will prevent developers from doing things accidentally and prod code doesn't really need to be so strict.

Reply:
Changed it typescript private only!